1-js/01-getting-started/4-devtools/article.md

```diff
@@ -1,64 +1,64 @@
 # Dasturchi konsoli
 
-Kodlarda xatoliklar ham bo'lish mumkin.Sizning yanglishishingiz ehtimoli ham anchagina. To'g'rirog'i, agr siz [robot] (https://en.wikipedia.org/wiki/Bender_(Futurama)) emas inson bo'lsangiz, albatta xatolar qilasiz.
+Kodlarda xatoliklar ham bo'lish mumkin. Sizning yanglishish ehtimolingiz ham biroz yuqori. To'g'rirog'i, agar siz [robot] (https://en.wikipedia.org/wiki/Bender_(Futurama)) emas, inson bo'lsangiz, albatta xatolar qilasiz.
 
-Lekin brauzerda xatolarni foydalanuvchilar to'gridan to'g'ri ko'rishmnaydi. Shuning uchun scriptda biror narsa yanglish ketsa, biz nima buzilganligini ko'rmaymiz va uni tuzata olmaymiz. 
+Lekin brauzerdagi xatolarni foydalanuvchilar to'gridan to'g'ri ko'rishmnaydi. Shuning uchun skriptda biror narsa yanglish ketsa, biz nima buzilganligini ko'rmaymiz va uni tuzata olmaymiz. 
 
-Xatoliklarni ko'rish va scriptlar haqida boshqa ma'lumotlar olish uchun brauzerlarga "developer tools" (dasturchi vositalari) o'rnatilgan. 
+Xatoliklarni ko'rish va skriptlar haqida boshqa ma'lumotlar olish uchun brauzerlarga "developer tools" (dasturchi vositalari) o'rnatilgan. 
 
-Ko'p dasturchilar loyihalash (dasturlash)da Chrome yoki Firefox suyanishadi, chunki ularda eng yaxshi dasturchi vositalari bor.Boshqa brauzerlar ham dasturlash vositasi bilan ta'minlay oladi, xatto ba'zida maxsus xususiyatlari ham mavjud, lekin ular Chrome yoki Firefoxga yeta olmaydi.Xullas, dasturchilarni ko'pining o'z "sevimli" brauzerlari bo'ladi va muammosi brauzeri bilan bog'liq bo'lgan holda boshqasiga o'tib ketadi.   
+Ko'p dasturchilar loyihalash (dasturlash)da Chrome yoki Firefoxdan foydalanishadi, chunki ularda eng yaxshi dasturchi vositalari bor. Boshqa brauzerlar ham dasturlash vositasi bilan ta'minlay oladi, hatto ba'zida o'zlarining maxsus xususiyatlari ham mavjud, lekin ular Chrome yoki Firefoxga yeta olmaydi. Xullas, ko'plab dasturchilarning o'z "sevimli" brauzerlari bo'ladi va brauzeri bilan bog'liq muammo bo'lgan holda boshqasiga o'tib ketadi.  
 
-Dasturlash vositalari kuchli ta'sirli bo'lib, ularning xususiyatlari ko'p. Dastlab biz ularni qanday ochishni, xatolarni ko'rib chiqishni hamda JavaScriptda buyruqlarni boshqarishni o'rganib chiqamiz. 
+Dasturlash vositalari kuchli ta'sirga ega bo'lib, ularning xususiyatlari ko'p. Dastlab biz ularni qanday ochishni, xatolarni ko'rib chiqishni hamda JavaScriptda buyruqlarni boshqarishni o'rganib chiqamiz. 
 
 ## Google Chrome
 
-[bug.html](bug.html) sahifasini ochasiz.
+[bug.html](bug.html) sahifasini oching.
 
 Bunda JavaScript kodida xatolik mavjud. Oddiy tashrif buyuruvchilar buni ko'rishmaydi, uni ko'rish uchun esa dasturchi vositalarini ochishimiz kerak bo'ladi. 
 
-`key:F12` tugmasini bosing, yoki agar Mac da bo'lsangiz, u holda `key:Cmd+Opt+J`.
+`key:F12` tugmasini bosing, yoki agar Macbook dan foydalanayotgan bo'lsangiz, u holda `key:Cmd+Opt+J`.
 
 Dasturchi vositalari kamchiliklar oynasini to'g'ridan to'g'ri ochib beradi. 
```

Comment:
The developer tools will open on the Console tab by default.

Dasturchi asboblari konsol yorlig'ini (tab) ochadi.

Reply:
balki yana mano ustida ishlarsiz, lekin oldingi qilingani xato